### PR TITLE
Precision Medicine use case (phase 1: Architecture and MVP)

### PR DESCRIPTION
**Pull request by Irtazaraza**
# Why is this PR needed?
closes: #18 
closes: #19 
closes: #20 
1) To implement to the architecture of BioGrakn
2) To identify the datasets applicable to the Precision Medicine usecase
2) To model the Grakn schema which maps the data entries to Grakn schema concepts
3) To implement the migration scripts to load data instances into grakn

# What does the PR do?
1) Create a Bazel project architecture for BioGrakn
2) Add data sets relevant to the minimum viable Precision Medicine use case
3) Includes the Precision Medicine Schema which models the respective use case
4) Create a migration class for each entity and relationship type to be migrated to the keyspace

# Does it break backwards compatibility?
Yes

# List of future improvements not on this PR
1) Add full coverage tests for source code
2) Write a detailed README.md file for BioGrakn
3) Add check style to project
4) Add licence headers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,8 +2,9 @@
 **/.DS_Store
```
Comment (lolski) [inline .gitignore]: Add ` bazel-*` in order to ignore temporary Bazel stuff like `bazel-bin`, `bazel-genfiles`, etc. Once you do, make sure to remove them as they already got added accidentally.

File: migrator/person/test/PersonTest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package grakn.biograkn.migrator.person.test;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class PersonTest {
```
Comment (lolski) [inline migrator/person/test/PersonTest.java]: This test isn't testing anything, right? Let's remove it

File: migrator/MigratorTest.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package grakn.biograkn.migrator;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MigratorTest {
+    @Test
+    public void exampleTest() {
+        assertEquals("migrator/test", "migrator/test");
```
Comment (lolski) [inline migrator/MigratorTest.java]: This test isn't testing anything, let's remove it.

File: migrator/person/test/PersonTest.java (hunk 2 above)
Comment (Irtazaraza) [reply migrator/person/test/PersonTest.java]: yeah. okay.

**Pull request merged**